react-carousel/src/state/selectors/carouselSelectors.js

```diff
@@ -0,0 +1,128 @@
+import { selector } from 'recoil';
+import _flow from 'lodash/flow';
```

Comment:
why do you names śtarting with an underscore as for `times`, `concat` or `flatten`, you have no underscore?

Reply:
I use _ to indicate this import comes from lodash library. I will change the rest of lodash imports so they will follow this pattern